lint/config.go

```diff
@@ -109,6 +109,16 @@ func ReadConfigsYAML(f io.Reader) (Configs, error) {
 	return c, nil
 }
 
+// MakeConfigsForAllPaths takes an input RuleConfig and applies it to all Rules and all file paths.
+func MakeConfigsForAllRulesAndPaths(rc RuleConfig) Configs {
```

Comment:
Do we need to make it public? Can we move it to linter_test.go?

Reply:
Yeah, we can move it in. I made it public because I figured it would be useful to other tests, but we can move it if we need it.

Reply:
I think it is better to make it private and only for tests, so we don't need to maintain it.